conjur-oss/README.md

```diff
@@ -101,31 +101,34 @@ The following table lists the configurable parameters of the Conjur OSS chart an
 |---------|-----------|-------|
 |`account`|Name of the Conjur account to be used by the Kubernetes authenticator|`"default"`|
 |`authenticators`|List of authenticators that Conjur will whitelist and load.|`"authn"`|
+|`conjurLabels`|Extra Kubernetes labels to apply to Conjur resources|`{}`|
 |`dataKey`|Conjur data key, 32 byte base-64 encoded string for data encryption.|`""`|
 |`databaseUrl`|PostgreSQL connection string. If left blank, a PostgreSQL deployment is created.|`""`|
-|`replicaCount`|Number of desired Conjur pods|`1`|
+|`deployment.annotations`|Annotations for Conjur deployment|`{}`|
 |`image.repository`|Conjur Docker image repository|`"cyberark/conjur"`|
 |`image.tag`|Conjur Docker image tag|`"latest"`|
 |`image.pullPolicy`|Pull policy for Conjur Docker image|`"Always"`|
 |`nginx.image.repository`|NGINX Docker image repository|`"nginx"`|
 |`nginx.image.tag`|NGINX Docker image tag|`"1.15"`|
 |`nginx.image.pullPolicy`|Pull policy for NGINX Docker image|`"IfNotPresent"`|
 |`persistentVolumeSize`|Size of persistent volume to be created for PostgreSQL|`"8Gi"`|
-|`storageClass`|Storage class to be used for PostgreSQL persistent volume claim|`nil`|
+|`postgres.image.pullPolicy`|Pull policy for postgres Docker image|`"IfNotPresent"`|
 |`postgres.image.repository`|postgres Docker image repository|`"postgres"`|
 |`postgres.image.tag`|postgres Docker image tag|`"10.1"`|
-|`postgres.image.pullPolicy`|Pull policy for postgres Docker image|`"IfNotPresent"`|
-|`deployment.annotations`|Annotations for Conjur deployment|`{}`|
-|`service.internal.type`|Conjur internal service type (ClusterIP and NodePort supported)|`"NodePort"`|
-|`service.internal.port`|Conjur internal service port|`443`|
+|`rbac.create`|Controls whether or not RBAC resources are created|`true`|
+|`replicaCount`|Number of desired Conjur pods|`1`|
+|`service.annotations`|Annotations for Conjur service|`{}`|
```

Comment:
This should be `service.internal.annotation` maybe (probably the original readme was bad)?